mucommander-command/src/main/java/com/mucommander/command/CommandManager.java

```diff
@@ -324,33 +301,31 @@ private static void buildFilter(FileFilter filter, AssociationBuilder builder) t
 
             attributeFilter = (AttributeFileFilter)filter;
             switch(attributeFilter.getAttribute()) {
-            case HIDDEN:
-                builder.setIsHidden(!attributeFilter.isInverted());
-                break;
+                case HIDDEN:
+                    builder.setIsHidden(!attributeFilter.isInverted());
+                    break;
 
-            case SYMLINK:
-                builder.setIsSymlink(!attributeFilter.isInverted());
-                break;
+                case SYMLINK:
+                    builder.setIsSymlink(!attributeFilter.isInverted());
+                    break;
```

Comment:
hmm, it looks OK for me and for idea formatter....

Reply:
it could be that idea accepts this, but for consistency there's [the formatter in our repo](https://github.com/mucommander/mucommander/blob/master/mucommander-code-format.xml) - I'm so used to use it that this indentation surprised me :)

Reply:
so, `case` and `switch` should be on the same level? Strange :)

however, I can reformat it if you like

Reply:
I'm ok with changing it to the way to formatted the switch-case statement but by changing the formatter, I do see value in having this formatter so we won't find PRs that are created by other IDEs having a lot of noise

Reply:
PR updated